src/dos/program_loadfix.cpp

```diff
@@ -77,6 +77,7 @@ void LOADFIX::Run(void)
 			// Use shell to start program
 			DOS_Shell shell;
 			shell.ExecuteProgram(filename, args);
+			shell.RunBatchFile();
```

Comment:
In reality, either ExecuteProgram or RunBatchFile runs, right? Can you put them in an if/else then to clarify how this works in actually reality?

The bare minimum would be an explanatory comment; right now, I'm looking at it and it's weird—execute a program first, then run a batch file? Huh? 😄

Reply:
They both need to run.  Like I said in the description up top, ExecuteProgram puts the batch file in a data structure and RunBatchFile pulls it off and runs it (it's a no-op if there's no batch file).

Reply:
Oh, right. Yes you did, indeed. It's so weird that I did not even comprehend it 😄 Yeah, please copy paste that top description into the code then, everybody will be super confused by this...

Next Windows build should be fine.

Reply:
I added a comment to this a couple days ago.  Was there anything else you wanted changed here?

Reply:
Thanks, that's all. Merge away 😎